src/Service.Tests/SqlTests/RestApiTests/Find/FindApiTestBase.cs

```diff
@@ -1098,6 +1098,47 @@ public async Task FindTestOnTableWithDatabasePolicy()
            );
         }
 
+        /// <summary>
+        /// Validates that Find API requests ignore predicates present in the request body
+        /// </summary>
+        [TestMethod]
+        public async Task FindApiTestWithPredicatesInRequestBody()
+        {
+            string requestBody = @"
+            {
+                ""$select"": ""id,title""
```

Comment:
select is not a predicate. It translates to a projection i.e. the SELECT clause of the query, can you please test with `$filter` as the predicate in the request body

Reply:
I've updated